go/oasis-node/integrationrunner/integration_test.go

```diff
@@ -16,4 +17,7 @@ func TestIntegration(t *testing.T) {
 	}
 
 	launch()
+
+	// Suppress coverage report on stdout.
+	_ = os.Stdout.Close()
```

Comment:
Do we only care about stdout?

Also, `os.Stdout = ioutil.Discard` might be slightly better behaved, though file descriptor re-use is unlikely.

Reply:
I wanted that too, but the types don't match.

Reply:
and yeah the issue that this fixes is only about stdout.

Reply:
Oh, it's an `os.File`, so something like:
```
  f, err := os.OpenFile(os.DevNull, os.O_RW, 0)
  if err != nil {
    t.Fatalf("Failed to open /dev/null") // Use testify if you want, whatever.
  }
  os.Stdout = f
```

Reply:
ok